enterprise-suite/values.yaml

```diff
@@ -23,3 +23,5 @@ imagePullPolicy: IfNotPresent
 prometheusDomain: prometheus.io
 # minikube debug resources
 minikube: true
+# expose-es-console
+exposeConsoleService: false
```

Comment:
gotta stop using vim man, or set vim to handle line endings properly.  see the little red note here, we see those in your files all the time

![image](https://user-images.githubusercontent.com/1753336/44602261-ef341880-a793-11e8-965c-df0fa548d3ed.png)

Reply:
this is n't vim fault, apparently there is a setting in intellij I didn't set 
No more from now on
Mac : Preferences > Editor > General > Ensure line feed at file end on save